src/_gettsim/groupings.py

```diff
@@ -93,8 +154,8 @@ def fg_id_numpy(  # noqa: PLR0913
 
             if (
                 child_hh_id == current_hh_id
-                # TODO (@hmgaudecker): Add correct conditions for grown up children
-                # https://github.com/iza-institute-of-labor-economics/gettsim/pulls/509
+                # TODO (@MImmesberger): Check correct conditions for grown up children
```

Comment:
Good occasion to [check the remark over there](https://github.com/iza-institute-of-labor-economics/gettsim/pull/509#issuecomment-1948171381) and close that PR.